ims/einstein/operations.py

```diff
@@ -217,11 +217,16 @@ def shutdown(self):
 
     # Provisions from HaaS and Boots the given node with given image
     @log
-    def provision(self, node_name, img_name, network, nic):
+    def provision(self, node_name, img_name, network, nic, hil_flag):
         try:
+            hil_bool = bool(hil_flag == "True")
```

Comment:
It is originally set as a boolean in the CLI, but when it gets passed to operations for some reason it gets converted in the background to a string, so this was the only way I could see to get it functioning appropriately.